libobs/audio-monitoring/osx/coreaudio-output.c

```diff
@@ -12,6 +12,8 @@
 
 #include "mac-helpers.h"
 
+volatile long audio_track_active;
```

Comment:
This variable should be static.
```suggestion
static volatile long audio_track_active;
```